hound_test.rb

```diff
@@ -0,0 +1,5 @@
+name = 'greg'
+
+puts name 
+puts 'something'
+puts 'hello world'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.